index.bs

```diff
@@ -323,7 +323,7 @@ below and in [[#index-defined-elsewhere]].
 : FIDO AppID
 :: The algorithms for [=determining the FacetID of a calling application=] and
     [=determining if a caller's FacetID is authorized for an AppID=] (used only in
-    the `appid` extension) are defined by [[!FIDO-APPID]].
+    the <code>[[#sctn-appid-extension|appid]]</code> extension) are defined by [[!FIDO-APPID]].
```

Comment:
I would \<dfn> "AppID" in the title for #sctn-appid-extension, and then just use [=AppID=] here.